docs/farming-&-staking/farming/additional-guides/gpu-plotter.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Experimental GPU Plotter
+sidebar_position: 4
+description: Tips on GPU Plotter
+keywords:
+    - Plotter
+    - Farming
+    - GPU
+---
+
+The first test builds of the **GPU Plotter** are now available!!!
+
+[Download GPU Plotter](https://github.com/autonomys/subspace/actions/runs/10850781628)
```

Comment:
Can we use the same TABS component for consistency? 

We can point directly at the executables, e.g. https://github.com/autonomys/subspace/actions/runs/10850781628/artifacts/1930245872

Reply:
This portion of the page is not visible.  I looked on other pages (including the one I copied from) and they all have this extra tab.